src/components/pages/containers/FormPageContainer.tsx

```diff
@@ -13,9 +24,136 @@ interface Props {
   onSubmit: Function;
   localePush: Function;
   tab: string;
+  step: number;
   application: ApplicationState;
 }
 
+const mapSteps = [
+  ['registered_boat', 'unregistered_boat', 'no_boat'],
+  ['private_person', 'company'],
+  ['overview']
+];
+
+const FormPageContainer = ({ selectedBerths, localePush, tab, application, ...rest }: Props) => {
+  const [step, setStep] = useState(0);
+  const [currTab, setTab] = useState('');
+  const [tabs, setTabs] = useState(['registered_boat', 'private_person', 'overview']);
+
+  useEffect(() => {
+    const currStep = Math.max(0, findIndex(mapSteps, s => s.includes(tab)));
+    setStep(currStep);
+    setTab(tab || mapSteps[currStep][0]);
+  });
+
+  const steps = [
```

Comment:
should export this array, also same for winter if possible

Reply:
I will fix it under this ticket [VEN-82](https://helsinkisolutionoffice.atlassian.net/browse/VEN-82)